ros/lib/app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+from ros.lib.models import db
+from ros.lib.config import DB_URI
+
+# Since we're using flask_sqlalchemy, we must create the flask app in both processor and web api
+app = Flask(__name__)
+# Initalize database connection
+app.config['SQLALCHEMY_DATABASE_URI'] = DB_URI
+db.init_app(app)
```

Comment:
should `app.py` also live inside `lib`?

Reply:
It is. I modified it to use local imports